### DISCUSSION TITLE
15sec delay while streaming on TV

### DISCUSSION THREAD

**Discussion by luberk8** [mrlt8/docker-wyze-bridge#482] [Q&A]
I am trying to create an automation that will start a stream on my TV, which can be addressed through its native roku interface or the connected nvidia shield. Starting the stream works fine, but the stream has a delay of about 15sec, which is too much for what I want to use it for.
On the home assistant web interface the delay is roughly 5sec and on the docker-wyze-bridge dashboard there is almost no delay.
Any ideas how I can get it to work on my TV?

**Comment by mrlt8** [comment]
Try to use the RTSP stream if you can as that will have the least delay.

Also, check the cache level on your media player - some players will try to buffer the video to allow to prevent the video from stopping.

**Reply by luberk8** [reply]
I had to remove docker_wyze_bridge since it was messing with my wifi too much. It went so far and basically made my Arlo camera system useless that is using the same wifi channel. I assume the delays are related to the same wifi issue

**Reply by mrlt8** [reply]
The wyze cameras tend to flood the network with UDP packets, so you may need to make some adjustments on your router.
